includes/UserMailer.php

```diff
@@ -760,53 +758,34 @@ private function composeCommonMailtext() {
 
 		$keys['$PAGEEDITOR_WIKI'] = $this->editor->getUserPage()->getCanonicalUrl();
 
-		# <Wikia>
-		// RT #1294 Bartek 07.05.2009, use the language of the wiki
-		$summary = ($this->summary == '') ? wfMessage( 'enotif_no_summary' )->inContentLanguage()->plain() : '"' . $this->summary . '"';
-		# </Wikia>
-		
-		# Replace this after transforming the message, bug 35019
+		$summary = ( $this->summary == '' ) ? wfMessage( 'enotif_no_summary' )->inContentLanguage()->plain() : '"' . $this->summary . '"';
```

Comment:
Parens are unnecessary here.  I realize you just added spacing, but its a pet peeve ;)